api/middleware/authRequired.js

```diff
@@ -7,18 +7,22 @@ const oktaJwtVerifier = new OktaJwtVerifier(oktaVerifierConfig.config);
  * A simple middleware that asserts valid access tokens and sends 401 responses
  * if the token is not present or fails validation.
  */
-const authRequired = async (req, res, next) => {
-  try {
-    const authHeader = req.headers.authorization || '';
-    oktaJwtVerifier
-      .verifyAccessToken(authHeader, oktaVerifierConfig.expectedAudience)
-      .then(async () => {
-        next();
-      })
-      .catch(next);
-  } catch (err) {
-    next(createError(401, err.message));
```

Comment:
Was able to verify that this piece of middleware data was not getting JSON WEB TOKEN from okta, thus not going through JWT Verification. With corrected code Web token is being passed and verified when calling on dashboard/incidents end points and passing correct response.

Reply:
This backend middleware was not operating in the desired manner. It failed to properly verify and pass the JWT. This code remedies that. The match/bearer line where you use regex was really interesting. My visual side would have liked for you to have shown what match returns. I think this was a really great job of isolating an issue, digging into it, and emerging with a seemingly solid solution.